garak/resources/fixer/20250224_lightweight_probe_defaults.py

```diff
@@ -0,0 +1,345 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+from garak.resources.fixer import Migration
+from garak.resources.fixer import _plugin
+
+
+class RenameFigstep_1(Migration):
+    def apply(config_dict: dict) -> dict:
+        """Rename FigStep probes to make lightweight the default"""
+
+        path = ["plugins", "probes", "visual_jailbreak"]
+        old = "FigStep"
+        new = "FigStepFull"
+        return _plugin.rename(config_dict, path, old, new)
+
+
+class RenameFigstep_2(Migration):
+    def apply(config_dict: dict) -> dict:
+        """Rename FigStep probes to make lightweight the default"""
+
+        path = ["plugins", "probes", "visual_jailbreak"]
+        old = "FigStepTiny"
+        new = "FigStep"
+        return _plugin.rename(config_dict, path, old, new)
```

Comment:
Example of consolidated rename:
```suggestion
class RenameFigstep(Migration):
    def apply(config_dict: dict) -> dict:
        """Rename FigStep probes to make lightweight the default"""

        path = ["plugins", "probes", "visual_jailbreak"]
        old = "FigStep"
        new = "FigStepFull"
        interim_config = _plugin.rename(config_dict, path, old, new)
        
        path = ["plugins", "probes", "visual_jailbreak"]
        old = "FigStepTiny"
        new = "FigStep"
        return _plugin.rename(interim_config, path, old, new)


```

or 
```suggestion
class RenameFigstep(Migration):
    def apply(config_dict: dict) -> dict:
        """Rename FigStep probes to make lightweight the default"""

        renames = [
            (["plugins", "probes", "visual_jailbreak"], "FigStep", "FigStepFull"),
            (["plugins", "probes", "visual_jailbreak"], "FigStepTiny", "FigStep"),
        ]
        interim_config = config_dict
        for path, old, new in renames:
            interim_config = _plugin.rename(interim_config, path, old, new)

        return interim_config
```

Reply:
tight, will amend